template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/TemplatePreparationObject.java

```diff
@@ -53,6 +53,8 @@
 
     private final Map<String, RDSConfig> rdsConfigs;
 
+    private final String rdsSslCertificateFile;
```

Comment:
Can we name it {{rdsSslCertificateFilePath}} or ending with location? I think it would be more straightforward that way.

Reply:
Good point. Fixed.